axonserver/src/main/java/io/axoniq/axonserver/localstorage/file/EventIterator.java

```diff
@@ -41,18 +41,18 @@ public EventInformation next() {
         return eventsInTransaction;
     }
 
-    public Long getTokenAfter(long instant) {
+    public Long getTokenAt(long instant) {
         if (hasNext()) {
             EventInformation event = next();
             if (event.getEvent().getTimestamp() <= instant) {
```

Comment:
if the timestamp is equal to instant, we should return the same token, not the next one. So I think that in this clause we need < i.o. <=